### DISCUSSION TITLE
Oracle DDL script doesn't have indexes

### DISCUSSION THREAD

**Discussion by iluwa** [kagkarlsson/db-scheduler#502] [Q&A] [answered]
Hi, I noticed that there are no indexes on the columns (execution_time, last_heartbeat) for the [Oracle DB example](https://github.com/kagkarlsson/db-scheduler/blob/master/db-scheduler/src/test/resources/oracle_tables.sql) while they exist for other DBs. 
Is this intentional?

**Comment by kagkarlsson** [comment, marked as answer]
No, the Oracle schema just have not gotten any love. Feel free to contribute the index-definitions :).  (the schema was contributed)
